code/game/objects/items/trash_vr.dm

```diff
@@ -26,8 +26,290 @@
 			user.drop_item()
 			var/belly = R.vore_selected
 			var/datum/belly/selected = R.vore_organs[belly]
-			src.forceMove(R)
+			forceMove(R)
 			selected.internal_contents |= src // Too many hoops and obstacles to stick it into the sleeper module.
 			R.visible_message("<span class='warning'>[user] feeds [R] with [src]!</span>")
 			return
 	..()
+
+/obj/item/device/flashlight/flare/attack(mob/living/M as mob, mob/living/user as mob)
```

Comment:
The copypasta is real. You can't just override attack on every object you want to eat.

Attacks are supposed to return a value, which you don't, when you call parent or handle it yourself, so this will probably break some of these objects.

There's no M == user check, and this is called before parent, so if you, for example, tried to hit someone with a broken bottle, they would be instantly fed the broken bottle instead of being attacked by it if they had the trait. This also makes it impossible to drink from a can, tea, coffee, dry ramen, etc as far as I can tell, or feed any of those to anyone else.